gems/safemode/CVE-2016-3693.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: safemode
+cve: 2016-3693
+title: Information leakage through templates
+date: 2016-04-20
+url: http://seclists.org/oss-sec/2016/q2/119
+description: |
+  A provisioning template containing `inspect` will expose sensitive
```

Comment:
@reedloden suggestion:

"Safemode is initialised with an optional 'delegate' object.
If the delegated object is a Rails controller, 'inspect' could
be called which then exposes all informations about the App,
including routes, secret tokens, caches and so on."